lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/validation/Gpg45DcmawValidator.java

```diff
@@ -1,21 +1,31 @@
 package uk.gov.di.ipv.core.library.domain.gpg45.validation;
 
 import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.domain.ContraIndicatorScores;
 import uk.gov.di.ipv.core.library.domain.gpg45.domain.CredentialEvidenceItem;
+import uk.gov.di.ipv.core.library.helpers.VcHelper;
+
+import java.util.Map;
 
 public class Gpg45DcmawValidator {
     @ExcludeFromGeneratedCoverageReport
     private Gpg45DcmawValidator() {
         throw new IllegalStateException("Utility class");
     }
 
-    public static boolean isSuccessful(CredentialEvidenceItem item) {
-        if (item.getCi() == null || item.getCi().isEmpty()) {
-            if (item.getFailedCheckDetails() != null) {
-                return false;
+    public static boolean isSuccessful(
+            CredentialEvidenceItem item,
+            Map<String, ContraIndicatorScores> ciScoresMap,
+            int ciScoreThreshold) {
+        if (item.getFailedCheckDetails() != null) {
+            return false;
+        }
+        if (item.getValidityScore() != 0) {
+            if (item.getCi() == null || item.getCi().isEmpty()) {
```

Comment:
I think we can get rid of the ci scoring checks here now.

Should double check with @kerrr how we want to handle the "success" status of the audit events. So when we retrieve a new VC, we sent an audit event that contains that VC's success status that uses this code. For the audit message is it fine to exclude CI checks in that status as well, or for the audit message do we want to mark it as not successful if it contains a CI?

Reply:
🤔 We shouldn't do CI scoring here. Even if we think that having a CI or not has some basing on success it's not the same issue as the scoring of CIs. As such it doesn't belong in this PR. So just leave this change out.

Reply:
I think a change might still need to be made potentially. This code is being called to trigger when a VC is considered successful or not during the select-cri lambda. Which probably doesn't need to count CI's into that fact anymore. We previously had it return false if a CI was present unless it was the Fraud CRI specifically with A01

Reply:
I've updated this now so we only check for presence of CIs for the VC success check in the retrieve-cri-credential lambda which uses it for the audit log. When we record VC success in the select-cri lambda we now ignore the CI check so the journey can continue.